claat/proto-renderer/html/templates-tests/inlinecode_test.go

```diff
@@ -0,0 +1,54 @@
+package htmltests
+
+import (
+	"testing"
+
+	"github.com/googlecodelabs/tools/claat/proto-renderer/html"
+	"github.com/googlecodelabs/tools/claat/proto-renderer/testing-utils"
+)
+
+func TestRenderInlineCodeTemplateEscaping(t *testing.T) {
+	tests := []*testingUtils.RendererTestingBatch{
+		{
+			testingUtils.NewInlineCode("< less-than"),
+			"<code>&lt; less-than</code>",
+			true,
+		},
+		{
+			testingUtils.NewInlineCode("> greater-than"),
+			"<code>&gt; greater-than</code>",
+			true,
+		},
+		{
+			testingUtils.NewInlineCode("/ backslash"),
+			"<code>/ backslash</code>",
+			true,
+		},
+		{
+			testingUtils.NewInlineCode(`\ forwardslash`),
+			`<code>\\ forwardslash</code>`,
+			true,
+		},
+		{
+			testingUtils.NewInlineCode("& ampersand"),
+			"<code>&amp; ampersand</code>",
+			true,
+		},
+		{
+			testingUtils.NewInlineCode(`" quotation`),
+			"<code>&#34; quotation</code>",
+			true,
+		},
+		{
+			testingUtils.NewInlineCode("' apostrophe"),
+			"<code>&#39; apostrophe</code>",
+			true,
+		},
+		{
+			testingUtils.NewInlineCode("{ Αα Ββ Γγ Δδ Εε Ϝϝ Ζζ Ηη Θθ Ιι Κκ Λλ Μμ Νν Ξξ Οο Ππ Ρρ Σσς Ττ Υυ Φφ Χχ Ψψ Ωω }"),
+			"<code>{ Αα Ββ Γγ Δδ Εε Ϝϝ Ζζ Ηη Θθ Ιι Κκ Λλ Μμ Νν Ξξ Οο Ππ Ρρ Σσς Ττ Υυ Φφ Χχ Ψψ Ωω }</code>",
+			true,
+		},
+	}
+	testingUtils.CanonicalRenderingTestBatch(html.Render, tests, t)
+}
```

Comment:
Error here will look the name per https://github.com/googlecodelabs/tools/pull/288#discussion_r296940674, changed accordingly for their input/output

Reply:
Let's add the field names here too